### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by bigkevmcd**
**What type of PR is this?**

> /kind cleanup

* Add GitLab token generation
* Simplify where to get the Buildah task
* Try and standardise the links to the latest release.
* Simplify the FAQ

**What does this PR do / why we need it**:

Tidies up some documentation, provides a nicer looking link to the Buildah task (rather than a Gist).

**Have you updated the necessary documentation?**

See https://github.com/rhd-gitops-example/docs

[ ] Updated

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -35,37 +40,43 @@ The following steps will guide you to setup GitOps for application delivery:
...
 
-Giving users the ability to simply modify the pipelines to their own needs at any point to a more customized approach. A simple push to the GitOps repo can alter the configuration of the GitOps repo and hence the style by which you want to run your organisation, by triggering a CI pipeline , further a merge to master triggers a deployment that further changes the state of your application on the cluster. Giving teams to use the power of OpenShift/Kubernetes to run powerful applications.
+Now's your chance to ramp up your organization using our modernized approach for end-to-end application delivery.
 
-Now we have an a team that set out as a start up but went on to be a full fledged organization with a multiple verticals and business set up in different geographical locations leveraging the use of OpenShift and GitOps with "Git as the source of truth".
+A simple GitOps configuration can be seen [here](https://github.com/rhd-gitops-example/gitops), all achieved with a list of commands listed in [gitops Command Reference](./commands), why wait?
```
Comment (wtam2018) [inline docs/README.md:77]: New location for example gitops repo: https://github.com/redhat-developer/gitops-repo-example

File: docs/commands/bootstrap/README.md [hunk 2]
```diff
@@ -30,25 +30,6 @@ $ gitops bootstrap
...
-| --service-webhook-secret              | Optional. Provide a secret that we can use to authenticate incoming hooks from your Git hosting service for the GitOps repository. (if not provided, it will be auto-generated)|
-| --overwrite                           | Optional. Overwrites previously existing GitOps configuration (if any) (default false) |
-| --sealed-secrets-ns string            | Optional. Namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator (default "cicd") |
-| --sealed-secrets-svc string           | Optional. Name of the Sealed Secrets Services that encrypts secrets (default "sealedsecretcontroller-sealed-secrets"") |
-| --git-host-access-token string  | Optional. Used to authenticate repository clones, and commit-status notifications (if enabled)|
-| --private-repo-driver string          | Optional. If your Enterprise Git repositories are on a custom domain, please indicate which driver to use github or gitlab|
-| --commit-status-tracker bool          | Optional. Enable or disable the commit-status-tracker which reports the success/failure of your pipelineruns to GitHub/GitLab|
-
```
Comment (wtam2018) [inline docs/commands/bootstrap/README.md:51]: I'd probably keep flag descriptions until we have a way to automatically generate docs for them.

File: docs/journey/day1/prerequisites/argocd.md [hunk 3]
```diff
@@ -1,10 +1,9 @@
+### ArgoCD Installation - with operator (Recommended)
 
-### ArgoCD Installation - with operator( Recommended )
-
-Create argocd namespace to install the operator
+Create argocd namespace to install the operator:
 
 ```shell
-oc create namespace argocd
+$ oc create namespace argocd
```
Comment (wtam2018) [inline docs/journey/day1/prerequisites/argocd.md:6]: Command line without a dollar sign make it easy to be copy and pasted.

Comment (bigkevmcd) [reply docs/journey/day1/prerequisites/argocd.md:6]: Yeah, but it's a very widely used convention.

The OpenShift documentation https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html uses the same convention

**Review by wtam2018 — DISMISSED** [review]

**Review by wtam2018 — APPROVED** [review]

**Pull request merged**